server/routes/account_management.py

```
@@ -119,19 +120,47 @@ def _resolve_row(row):
             ORDER BY CASE WHEN user_id = %s THEN 0 ELSE 1 END
             """,
             (user_id, org_id, user_id),
         )
 
-        for provider, account_id, role_arn, last_verified in cursor.fetchall():
+        gcp_accounts_list: list = []
+        for (
+            provider,
+            account_id,
+            role_arn,
+            last_verified,
```

Ruff (0.15.14)

[warning] Loop control variable `last_verified` not used within loop body

Rename unused `last_verified` to `_last_verified`

(B007)